### ISSUE TITLE
Update try.tarantool.org boxes

### ISSUE THREAD

**Issue by Totktonada** [tarantool/doc#1450]
* https://www.tarantool.io/en/try-dev/ is on 2.2.1-3-g878e2a42c (2.5.1 is released now).
* https://www.tarantool.io/en/try-lts/ is on 1.10.3-0-g0b7078a93 (1.10.7 is released now).

Please, update them.

Also, I see 'stable' and 'lts' in the dropping versions list and 'stable' is 'dev'. It is confusing ('dev' == 'stable', that's strange). Let's use two terms: say, 'latest' (for a last released beta, 2.5.1 now) and 'LTS' (for a last release from LTS series, 1.10.7 now).

'latest' is prefereble over 'dev' or 'stable', because is consistent with [/latest/](https://github.com/tarantool/doc/issues/453) documentation URLs.

So, in short, I would do the following.

* Move `/try-dev/` to `/try-latest/`, setup redirect from the old URL.
* Update `/try-latest/` to 2.5.1.
* Update `/try-lts/` to 1.10.7.
* Update 'stable' dropping list item to 'latest'.
* Update 'lts' dropping list item to 'LTS'.

Alternatively we can proceed with the minimal changes:

* Update `/try-dev/` to 2.5.1.
* Update `/try-lts/` to 1.10.7.
* Update 'stable' dropping list item to 'dev'.

**Comment by kyukhin** [comment]
Can we add bleeding version as well on try.? /try-master?

**Comment by Totktonada** [comment]
I guess a last beta is enough for a user if it'll be updated right after a release.

However #1453 + #1454 may require this.

**Comment by Totktonada** [comment]
@artembo https://try.tarantool.io is on track now AFAIK. So the issue becomes obsolete, correct?
